client/firewall/create_linux.go

```diff
@@ -58,6 +59,12 @@ func NewFirewall(context context.Context, iface IFaceMapper) (firewall.Manager,
 		log.Info("no firewall manager found, trying to use userspace packet filtering firewall")
 	}
 
+	if fm != nil {
+		if err := fm.Init(stateManager); err != nil {
```

Comment:
What will be if the **iface.IsUserspaceBind()** scope happens an error? Should not we revert this initialization?

Reply:
Probably, but this is out of scope for the PR. I just splitted out the init logic in the `Create` functions so it is not run when cleaning up only.